R/sfi_plot_laqueur_2.R

```diff
@@ -15,163 +15,192 @@ sfi_plot_laqueur_2 <- function(){
     geom_smooth(method = 'loess',
-                size = 0.5, 
                 alpha = 0.6,
-                color = 'black',
-                linetype = 2) + 
-    geom_point(size = 3, 
-               alpha = 0.6) +
+                fill = 'grey',
+                linetype = 0) + 
+    geom_point(size = 25, 
+               color = 'black',
+               fill = 'black',
+               alpha = 0.6,
+               pch = 21) +
+    geom_text(aes(label = paste0(percent_of_conducted_hearings_resulting_in_a_grant, 
+                                 '%')),
+              size = 5,
+              color = 'grey',
+              nudge_y = 0) +
+    theme(legend.position = 'none') +
+    xlim(c(2007, 2015)) + ylim(c(0, 50)) +
     labs(x = 'Year',
-         y = 'Rate of parole grant',
+         y = 'Hearings resulting in grant',
          title = 'Version 5',
-         caption = 'Fitted line smoothed with a local regression') +
-    theme_sfi()
-  
+         caption = paste0('Smoothed with a local regression', '\n', 
+                          'with bands representing standard errors')) +
+    theme_sfi(lp = 'none')
+
   
   # version 6
-  g6 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
-    geom_bar(stat = 'identity', 
-             alpha = 0.8,
-             fill = 'white',
-             color = 'black') +
-    geom_text(aes(label = paste0(percent_of_conducted_hearings_resulting_in_a_grant, ' %')),
-              size = 3, 
-              color = 'black',
-              nudge_y = 1) +
+  g6 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = percent_of_conducted_hearings_resulting_in_a_grant)) +
+    geom_smooth(method = 'loess',
+                alpha = 0.7,
+                fill = 'black',
+                linetype = 0) + 
+    geom_point(size = 4, 
+               color = 'black',
+               fill = 'grey',
+               alpha = 0.8,
+               pch = 21) +
     labs(x = 'Year',
-         y = 'Rate of parole grant',
-         title = 'Version 6') +
+         y = 'Hearings resulting in grant',
+         title = 'Version 6',
+         caption = paste0('Smoothed with a local regression', '\n', 
+                          'with bands representing standard errors')) +
     theme_sfi()
   
-  
   # version 7
-  g7 <- ggplot(data, 
-               aes(x = year, 
-                   y = percent_of_conducted_hearings_resulting_in_a_grant)) +
+  g7 <- 
+    ggplot(data, 
+           aes(x = year, 
+               y = percent_of_conducted_hearings_resulting_in_a_grant)) +
     geom_bar(stat = 'identity', 
              alpha = 0.8,
-             fill = 'grey',
-             color = 'black') +
-    geom_text(aes(label = paste0(percent_of_conducted_hearings_resulting_in_a_grant, ' %')),
-              size = 3, 
-              alpha = 0.6,
-              nudge_y = 1) +
+             fill = 'black',
+             color = 'grey') +
```

Comment:
I suggest losing the "color" argument here. The bars don't need a border...

Reply:
Ok cool